src/components/Game/Game.tsx

```diff
@@ -31,6 +66,12 @@ export const Game = ({ game }: GameProps) => {
 
   return (
     <div>
+      <Toaster
+        position="top-center"
+        toastOptions={{
+          style: { marginTop: '3rem', background: '#F4F4F4', border: '1px solid #333333' },
```

Comment:
You can give classnames here instead of inlineStyling. Especially if different toasts need to be styled differently, see figma.